charts/apisix/Chart.yaml

```diff
@@ -31,7 +31,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.1.0
+version: 1.1.1
```

Comment:
I think we shouldn't bump the APISIX chart's version.

We can bump this version after this PR is merged.
Then open a new PR to update dependencies.